modular_skyrat/modules/novaya_ert/code/toolbox.dm

```diff
@@ -0,0 +1,57 @@
+/obj/item/storage/toolbox/ammo
```

Comment:
Why... Is this a subtype of toolboxes, exactly?

Reply:
> Why... Is this a subtype of toolboxes, exactly?

Ask /tg/, lmao.